hilti/toolchain/src/compiler/codegen/types.cc

```diff
@@ -208,11 +216,16 @@ struct VisitorDeclaration : hilti::visitor::PreOrder<cxx::declaration::Type, Vis
                         cg->popSelf();
                     }
 
-                    auto x = cxx::declaration::Local{cxx::ID(f.id()),
-                                                     t,
-                                                     {},
-                                                     default_,
-                                                     (f.isStatic() ? "inline static" : "")};
+                    cg->popCxxBlock();
+
+                    if ( default_ )
+                        ctor.addStatement(fmt("%s = %s", cxx::ID(f.id()), *default_));
```

Comment:
`%s = std::move(%s)`?

Reply:
I would prefer to not move the value here since even with a moved value we generate the same code (`fmt` should bind a const ref), but could introduce e.g., use-after-move issues should we ever do more with `default_` in this scope.